src/igvfd/types/sample.py

```diff
@@ -451,6 +470,8 @@ class PrimaryCell(Biosample):
     schema = load_schema('igvfd:schemas/primary_cell.json')
     embedded_with_frame = Biosample.embedded_with_frame
     audit_inherit = Biosample.audit_inherit
+    set_status_up = Biosample.set_status_up + []
```

Comment:
this is a general comment on samples - i think we should add "Treatment" to the "up" lists